ytxp-plutarch/test/Cardano/YTxP/Test/Control/Yielding/Scripts.hs

```diff
@@ -0,0 +1,33 @@
+{-# OPTIONS_GHC -Wno-deferred-out-of-scope-variables #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+module Cardano.YTxP.Test.Control.Yielding.Scripts (tests) where
+
+import Cardano.YTxP.SDK.SdkParameters (AuthorisedScriptsSTCS (AuthorisedScriptsSTCS))
+import Cardano.YTxP.Test.Control.Yielding.Scripts.Attacks (testAttacksR)
+import Cardano.YTxP.Test.Control.Yielding.Scripts.NominalCases (testNominalCasesR)
+import Cardano.YTxP.Test.Control.Yielding.Scripts.Utils (
+  ScriptsTestsParams (
+    ScriptsTestsParams,
+    authorisedScriptHash,
+    authorisedScriptsManagerHash,
+    authorisedScriptsSTCS
+  ),
+ )
+import Control.Monad.Reader (Reader, runReader)
+import Test.Tasty (TestTree, testGroup)
+
+dummyParams :: ScriptsTestsParams
+dummyParams =
+  ScriptsTestsParams
+    { authorisedScriptHash = "22222222222222222222222222222222222222222222222222222222"
+    , authorisedScriptsSTCS = AuthorisedScriptsSTCS "bb"
```

Comment:
note that you won't get a run-time error when `show`ing this, but its still not well-formed. It would need to be updated if we get full phase-1 validation wired in to PCB